setup.py

```diff
@@ -15,7 +15,8 @@
     author="FilSwan",  # Placeholder for author's name
     author_email="author@email.com",  # Placeholder for author's email
     install_requires=["web3==5.31.1", "requests==2.28.1", "requests_toolbelt==0.10.1"],
-    packages=["swan", "swan.common", "swan.api"],
+    packages=find_packages(where="src"),  # Automatically find packages in src
```

Comment:
maybe we should only make certain folders available, not sure if this is the best idea to give access to all folders

Reply:
@DanielJiangCloud 
we can exclude some of them, but the directory is only `src` it means only swan package'